.github/workflows/linter.yml

```diff
@@ -41,4 +41,6 @@ jobs:
       - name: Run mypy
         run: |
           pip install mypy
-          mypy .
+          pip install -r python/requirements.txt
+          mypy --config-file=.github/linters/mypy.ini run_ingestion/ python/ingestion/
```

Comment:
Oh, your comment in the standup reminded me - I think we should use the "--disallow-untyped-defs" option. I wouldn't block this PR on it because enabling it would require going through all existing code and adding types, but I think we should do this before investing more heavily in the codebase.

What do you think?

Reply:
Addressed above, tldr; I agree 100%